ui/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -306,6 +311,7 @@ export default class ImportWithSeedPhrase extends PureComponent {
           }
           error={confirmPasswordError}
           autoComplete="new-password"
+          passwordrules="minlength: 8; maxlength: 20; required: lower; required: upper; required: digit;"
```

Comment:
I don't believe `passwordrules` is a widely supported attribute, and I believe all of these requirements can be achieved via `minLength`, `maxLength`, and `pattern` attributes.

Reply:
@darkwing minLength, maxLength and pattern attributes are created and added.
You can see changes here: c58db5fbcc2d8fd7680e2ad17a66d0a81015c642 and dd4811f34029a47e9ead01fd5d4da45ea748f12b
Is it okey now?